tests/application/test_connect.py

```diff
@@ -201,15 +200,12 @@ async def test_disconnect_failure(device, make_application):
 
 
 @pytest.mark.parametrize("device", [FormedLaunchpadCC26X2R1])
+@patch("zigpy_znp.zigbee.application.ControllerApplication._watchdog_period", new=0.1)
 async def test_watchdog(device, make_application):
     app, znp_server = make_application(server_cls=device)
-    await app.startup(auto_form=False)
-
     app._watchdog_feed = AsyncMock(wraps=app._watchdog_feed)
 
-    with patch("zigpy.application.ControllerApplication._watchdog_period", new=0.1):
-        await asyncio.sleep(0.6)
```

Comment:
uh, I did not want to remove this. I'll add it back, even though it seems like the `startup` takes long enough now.